### PR TITLE
Table menu

### PR DESCRIPTION
**Pull request by wpasadilla**

### CODE REVIEW

File: server/report.20200413.183036.14372.0.001.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
+
```
Comment (aidan-lee) [inline server/report.20200413.183036.14372.0.001.json:1]: What is this file?

Comment (aidan-lee) [inline server/report.20200413.183036.14372.0.001.json]: What is this file?

File: server/report.20200413.184125.11992.0.001.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
+
```
Comment (aidan-lee) [inline server/report.20200413.184125.11992.0.001.json]: What is this file?

File: server/report.20200413.184214.4480.0.001.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
+
```
Comment (aidan-lee) [inline server/report.20200413.184214.4480.0.001.json]: What is this file?

File: server/utilities/PokerUtils.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /**
  * The number of seats at a table
  */
-const NUM_OF_TABLE_SEATS = 6;
+const NUM_OF_TABLE_SEATS = 1;
```
Comment (darbrius) [inline server/utilities/PokerUtils.js]: Why was this changed, this is a constant stating the maximum number of people that can be playing at a table?

File: poker-pals/src/components/Tables.vue [hunk 5]
```diff
@@ -23,7 +23,23 @@
...
             </div>
             <div class = "seats" id = "seatsId">
-            <!-- TODO: Update seat availability as seats fill up.--> 
+                <p> {{room.numberOfFullSeats}}/{{room.numberOfTableSeats}}</p>
+            </div>
+        </div>
+      </div>
+      <!-- Display full table --> 
+        <!-- Join room on click --> 
+      <div class="table-full" id = "table-click" v-else v-on:click="joinRoom(room)">
```
Comment (rzayevan) [inline poker-pals/src/components/Tables.vue:33]: What does this `v-else` do?

File: server/utilities/PokerUtils.js (hunk 4 above)
Comment (wpasadilla) [reply server/utilities/PokerUtils.js]: It was changed to speed up testing and not require 6 players to test. Will change back to default 6.

File: poker-pals/src/components/Tables.vue (hunk 5 above)
Comment (wpasadilla) [reply poker-pals/src/components/Tables.vue:33]: v-else is used to change the appearance of the table once full.

Comment (rzayevan) [reply poker-pals/src/components/Tables.vue:33]: Nevermind, there's a `v-if` at the top

**Review by devongillis — APPROVED** [review]
I'll approve because they are not major issues, but first of all I think all tables should have a number in their name, that is the initial tables should contain "1" (right now the initial tables lack a number)

second, when a new table is added it should be placed beside the other tables of the same name (right now it just gets added to the bottom of the list)
